src/main/resources/userEmailTemplate-feedbackSession.html

```diff
@@ -61,9 +61,7 @@
     For any non-technical queries (e.g. assigned to the wrong team, misspelled name, request for deadline extension, etc.), you can contact your instructor(s) at: ${coOwnersEmails}.
 </p>
 
-<p>
-    If you encounter any problems when using the system, you can email TEAMMATES support team at ${supportEmail}.
-</p>
+${supportFragment}
```

Comment:
I believe it may be better to add this part in here:
```
<p>
    If you encounter any problems when using the system, you can email TEAMMATES support team at ${supportEmail}.
</p>
```
and add a field for any additional instructions before this section.